src/itinerary/Itinerary.tsx

```diff
@@ -50,6 +50,8 @@ const computeRootA11yProps = (ariaLabel?: string, ariaLabelledBy?: string): Root
 // Get places between departure and arrival
 const getIntermediatePlaces = (places: Place[]) => places.slice(1, -1)
 
+const stopoverCountToCollapseFrom = 2
```

Comment:
Consistency: 
- Put your const in UPPER_CAMEL_CASE
- More understandable to let it at 1 since the behavior is "no collapse if only one"

You'll need to change the code below:
`intermediatePlaces.length > stopoverCountToCollapseFrom`